### PR TITLE
Upgrade release

### PR DESCRIPTION
**Pull request by resdenia**
-   Update docker image, to upload playwright to the Docker.
-   CSS fix, update test status box.
-   Add cleanup for useEffect

### CODE REVIEW

File: service/lambdaFunction/index.js [hunk 1]
```diff
@@ -1,34 +1,56 @@
...
+			});
+			await context.tracing.start({ screenshots: false, snapshots: false });
+
+			page = await context.newPage();
+			
+	
+
+	const navigationPromise = page.waitForNavigation()
+
+await page.goto('https://logz.io/')
```
Comment (Doron-Bargo) [inline service/lambdaFunction/index.js]: Why you have this code inside the release

**Review by Doron-Bargo — CHANGES_REQUESTED** [review]
Please see my comment

File: service/lambdaFunction/index.js (hunk 1 above)
Comment (resdenia) [reply service/lambdaFunction/index.js]: @Doron-Bargo  Yeap, it should be removed. Test code from the previous testing in the lambda function itself.

**Review by Doron-Bargo — APPROVED** [review]
lgtm

**Pull request merged**